web/src/main/java/org/mskcc/cbio/oncokb/api/pvt/PrivateAnnotationController.java

```diff
@@ -0,0 +1,353 @@
+package org.mskcc.cbio.oncokb.api.pvt;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import org.apache.commons.collections.map.HashedMap;
+import org.apache.commons.lang3.StringUtils;
+import org.kohsuke.github.GHCompare.Tree;
+import org.mskcc.cbio.oncokb.config.annotation.PremiumPublicApi;
+import org.mskcc.cbio.oncokb.util.AlterationUtils;
+import org.mskcc.cbio.oncokb.util.CacheUtils;
+import org.mskcc.cbio.oncokb.util.EvidenceTypeUtils;
+import org.mskcc.cbio.oncokb.util.EvidenceUtils;
+import org.mskcc.cbio.oncokb.util.GeneUtils;
+import org.mskcc.cbio.oncokb.util.IndicatorUtils;
+import org.mskcc.cbio.oncokb.util.LevelUtils;
+import org.mskcc.cbio.oncokb.util.MainUtils;
+import org.mskcc.cbio.oncokb.util.TumorTypeUtils;
+import org.mskcc.cbio.oncokb.model.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+import java.util.Objects;
+
+@RestController
+@Api(tags = "Annotation")
```

Comment:
This should be Annotations so it can be included with other `annotate` endpoints.